overlord/devicestate/devicemgr.go

```diff
@@ -473,6 +473,11 @@ func (m *DeviceManager) ensureBootOk() error {
 		return nil
 	}
 
+	// XXX: we need a boot-ok for recovery mode
```

Comment:
we probably need a convention for this kind of XXX, given that is in old code I fear we will just forget about it

Reply:
Should probably not be skipped here but instead boot/snapstate.UpdateBootRevisions() should be updated.

Reply:
Or add "TODO:UC20"

Reply:
we agreed on TODO:UC20: